docs/vaahcms-2/backend/roles.md

```diff
@@ -0,0 +1,238 @@
+# Roles
+
+[[toc]]
+
+## Purpose
+A role is a group of permissions that allow a user who has been given the role to carry out specific tasks. For instance, the super administrator, who can be regarded as the application's owner, is permitted to carry out all tasks in the application. He or she can add users, remove or edit, and do other things.
+
+## Features & Demos
+
+#### Default Roles 
+
+VaahCMS gave you a few default roles that it built during application setup using seeds.
+
+| Roles    | Description                                                  |
+| ------------- | ------------------------------------------------------------ |
+| Super Administrator | The application's owner is super administrator. Users with super administrator privileges have full access and control over the data and application. |
+| Administrator | Users who have administration roles has all the permissions access and manage the data. Somebody who has access to all the administration features within the application. |
```

Comment:
@wri-tanmoy-d001 @we-vikram-wri231 you should also write what other roles can't do.

Reply:
@themodernpk Ok Sir
we will update